jaxopt/_src/quadratic_prog.py

```diff
@@ -85,8 +86,7 @@ def matvec(u):
   # Solves the following linear system:
   # [[Q A^T]  [primal_var = [-c
   #  [A 0  ]]  dual_var  ]    b]
-  return linear_solve.solve_cg(matvec, (minus_c, b), init=init_params,
-                               maxiter=maxiter)
+  return solve(matvec, (minus_c, b),tol=1e-07, maxiter=maxiter)
```

Comment:
Add a `tol` option to the function instead and use `1e-5` as default  for consistency. Add a space before the comma.

Reply:
Changed to 1e-5 in linear_solve.solve_gmres instead for consistency. Otherwise doesn't pass _check_derivative_A_and_b test

Reply:
I see! I think the right way to do it is to add `tol` as an attribute of `QuadraticProgramming` and then you can call `_solve_eq_constrained_qp(..., tol=self.tol)`. Then, in your test, you can instantiate `QuadraticProgramming(..., tol=1e-7)`.

Reply:
ok, done that!